condor-ap/gratia-probe-condor-ap.cron

```diff
@@ -0,0 +1 @@
+0,15,30,45 * * * * root /usr/share/gratia/common/cron_check /etc/gratia/condor-ap/ProbeConfig && /usr/share/gratia/condor-ap/condor_meter -s 900
```

Comment:
We can put this in a separate PR if you prefer but I'd really like to go the `SCHEDD_CRON` route and it appears that @rynge  is on board: https://github.com/opensciencegrid/osg-flock/blob/master/rpm/80-osg-flocking.conf#L56-L62